### PR TITLE
BXMSDOC-4118-master: Added a new chapter about coverage report in the test scenario document.

### PR DESCRIPTION
**Pull request by kaldesai**
[Epic](https://issues.jboss.org/browse/BXMSDOC-4007)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-4118)
[Test Scenario RHPAM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4118-CoverageReport-PAM/#test-scenarios-coverage-report-con)
[Test Scenario RHDM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4118-CoverageReport-DM/#test-scenarios-coverage-report-con)

Please review chapter 11. Coverage report for test scenarios

### CODE REVIEW

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[id='test-scenarios-coverage-report-dmn-based-proc']
+= Coverage report for DMN-based test scenarios
+
+Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+
+.Prerequisites
+* The DMN-based test scenario template has been created for the selected test scenario.For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: Typo - missing space:

s/.F/. F

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* The individual test scenarios have been defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
+. Open the DMN-based test scenario in the test scenario designer.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: Pluralize test scenarios designer:

s/test scenario designer/test scenarios designer

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[id='test-scenarios-coverage-report-dmn-based-proc']
+= Coverage report for DMN-based test scenarios
+
+Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+
+.Prerequisites
+* The DMN-based test scenario template has been created for the selected test scenario.For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+* The individual test scenarios have been defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: Question - Should there be a step or steps to help the user navigate to the test scenarios designer? I see most test scenario related docs do not have this.

Something like this example:
* Log in to {CENTRAL}.
* Click *Menu* -> *Design* -> *Projects*, then *{PROJECT}*.

Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: There are steps to help the user to the test scenarios designer. It is already defined in section 3.4 of the test scenario document.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: ack

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: Ack.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[id='test-scenarios-coverage-report-con']
+= Coverage report for test scenarios
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc]: ack

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,13 @@
+[id='test-scenarios-coverage-report-dmn-based-proc']
+= Coverage report for DMN-based test scenarios
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: Thank you for the IBMSG reference and also ack.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+
+.Prerequisites
+* The DMN-based test scenario template has been created for the selected test scenario.For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+* The individual test scenarios have been defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
+. Open the DMN-based test scenario in the test scenario designer.
+. Run the defined test scenarios.
+. Click `Coverage Report` on the right of the test scenario designer to display the test coverage statistics.
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: Ack.

**Pull request merged**